src/parser/jobs/rdm/modules/Procs.js

```diff
@@ -5,8 +5,40 @@ import {ActionLink, StatusLink} from 'components/ui/DbLink'
 import ACTIONS from 'data/ACTIONS'
 import STATUSES from 'data/STATUSES'
 import Module from 'parser/core/Module'
-import {TieredSuggestion} from 'parser/core/modules/Suggestions'
-import {PROCS, SEVERITY_MISSED_PROCS, SEVERITY_OVERWRITTEN_PROCS, SEVERITY_INVULN_PROCS} from 'parser/jobs/rdm/modules/ProcsEnum'
```

Comment:
With bringing this into the module directly, is there anything else using the ProcsEnum, or can that be deleted?

Reply:
It appears not, I had comments about Dualcast in there so I had ignored it.  I'll push another commit with it deleted here shortly.